src/rules/quotemarkRule.ts

```diff
@@ -75,18 +73,12 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 
     public isEnabled(): boolean {
-        return super.isEnabled() && (this.ruleArguments[0] === OPTION_SINGLE || this.ruleArguments[0] === OPTION_DOUBLE);
+        return super.isEnabled() && hasSingleDoublePreference(this.ruleArguments);
     }
 
     public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
         const args = this.ruleArguments;
-        if (args.length > 0) {
-            if (args[0] !== OPTION_SINGLE && args[0] !== OPTION_DOUBLE) {
-                showWarningOnce(`Warning: First argument to 'quotemark' rule should be "${OPTION_SINGLE}" or "${OPTION_DOUBLE}"`);
```

Comment:
It's funny that this code would show the correct warning, but was never executed because the rule was disabled.